internal/promoter/image/sign.go

```diff
@@ -46,6 +47,12 @@ const (
 	TestSigningAccount = "k8s-infra-promoter-test-signer@k8s-cip-test-prod.iam.gserviceaccount.com"
 )
 
+var rtripper *ratelimit.RoundTripper
```

Comment:
Maybe move both to round trip package so there's a single process-scoped global?

most ideal would be instancing once and passing to everywhere but that may require breaking changes etc...

Reply:
Unless the process authenticates with multiple accounts or something

Reply:
OK I've moved them to a `ratelimiter.Limiter` global :eyes: